plugins/social_network/social_network.cpp

```diff
@@ -220,7 +566,7 @@ namespace golos { namespace plugins { namespace social_network {
         if (itr != by_permlink_idx.end()) {
             return get_discussion(*itr, limit);
         }
-        return helper->create_discussion(author);
+        return helper->create_discussion(*itr);
```

Comment:
`itr` is equal to `.end()` . Why do you pass invalid object to create_discussion?

Reply:
There were another logic which returns empty discussion. Return it back.

Reply:
Fixed